etc/openqa/openqa.ini

```diff
@@ -136,3 +136,11 @@ blocklist = job_grab job_done
 # Specify the number of screenshot ID ranges (with a size as configured by screenshot_cleanup_batch_size) to process in a single Minion
 # job (reduce to avoid Minion jobs from running very long and possibly being interrupted, increase to reduce the number of Minion jobs)
 #screenshot_cleanup_batches_per_minion_job = 450
+
+[job_settings_ui]
+# Specify the keys of the test variables of the test suite settings which their values contain a file
+# to render individual paths in job settings as links.
+# Directories should be under CASEDIR root path or under the /data folder of the CASEDIR. /data folder is used as default
+# but it can be configured to cover needs of other test distribution repos. To change it, add the `default_data_dir` variable
+# with the name of the directory.
```

Comment:
The job settings don't need to come from a test suite. It also doesn't work for any test variable because if the test variable is only set on os-autoinst level it is not part of openQA's job settings. I would also rephrase it a little bit.

```suggestion
# Specify the keys of job settings which reference a file and should therefore be rendered
# as links to those files within the job settings tab.
# Directories should be under the `CASEDIR` root path or under the `data` folder of the `CASEDIR`. The `data`
# folder is used as default but it can be configured to cover needs of any test distribution. To change it, add the
#  `default_data_dir` variable with the name of the directory.
```